### PR TITLE
[RFC] docs: Remove some references to MS-DOS

### PR DESCRIPTION
**Pull request by sethjackson**
We don't support MS-DOS.

### CODE REVIEW

**Comment by equalsraf** [comment]
LGTM

File: runtime/doc/vi_diff.txt [hunk 1]
```diff
@@ -59,8 +59,6 @@ Support for different systems.
 	Vim can be used on:
 	- All Unix systems (it works on all systems it was tested on, although
 	  the GUI and Perl interface may not work everywhere).
-	- MS-DOS in real-mode (no additional drivers required).
-	- In protected mode on Windows 3.1 and MS-DOS (DPMI driver required).
 	- Windows 95 and Windows NT, with support for long file names.
```
Comment (hayd) [inline runtime/doc/vi_diff.txt]: is Windows 95 supported??

Comment (sethjackson) [inline runtime/doc/vi_diff.txt]: LOL no. But we need to fix a ton of Windows 95/98/ME/NT references too. So I left this line.

Comment (Pyrohh) [inline runtime/doc/vi_diff.txt]: :+1:

**Comment by Pyrohh** [comment]
Thanks all, merged with some additional removals.

@sethjackson in the future please put the PR message in the commit message.

**Comment by sethjackson** [comment]
> @sethjackson in the future please put the PR message in the commit message.

Will do.

**Pull request closed without merging**